source/user/installation_guide.rst

```diff
@@ -65,6 +65,52 @@ machine. In this case, the only system requirement is to have `docker`_,
 All the data and configuration needed to run a simulation are located in the `plug`_
 repository, along with usage examples.
 
+Natively
+--------
+
+The framework can be used natively after some initial setup. This first requires the following:
+
++ install julia 1.5 and create an environment as described in `REISE.jl`_
++ clone `PowerSimData`_ and `REISE.jl`_ (preferably in the same directly)
++ install the solver you wish to use within your julia environment (either Gurobi or GLPK)
+
+Create a virtualenv and activate it using the following. For the activate step, it's important
+to follow exactly (instead of using the "source" command) - this ensures that `sys.executable` will give back the python 
```

Comment:
I think what you want here is ``sys.executable``, i.e., verbatim, and in ReST it is activated with double quote \`\`